data/transactions/logic/opcodes.go

```diff
@@ -164,6 +166,28 @@ var OpSpecs = []OpSpec{
 
 	{0x70, "asset_holding_get", opAssetHoldingGet, assembleAssetHolding, disAssetHolding, twoInts, oneAny.plus(oneInt), 2, runModeApplication, opSize{1, 2, nil}},
 	{0x71, "asset_params_get", opAssetParamsGet, assembleAssetParams, disAssetParams, oneInt, oneAny.plus(oneInt), 2, runModeApplication, opSize{1, 2, nil}},
+
+	{0x72, "assert", opAssert, asmDefault, disDefault, oneInt, nil, 3, modeAny, opSizeDefault},
+	{0x73, "min_balance", opMinBalance, asmDefault, disDefault, oneInt, oneInt, 3, runModeApplication, opSizeDefault},
+
+	{0x74, "getbit", opGetBit, asmDefault, disDefault, twoInts, oneInt, 3, modeAny, opSizeDefault},
+	{0x75, "setbit", opSetBit, asmDefault, disDefault, threeInts, oneInt, 3, modeAny, opSizeDefault},
+	{0x76, "getbyte", opGetByte, asmDefault, disDefault, byteInt, oneInt, 3, modeAny, opSizeDefault},
+	{0x77, "setbyte", opSetByte, asmDefault, disDefault, byteIntInt, oneBytes, 3, modeAny, opSizeDefault},
+	{0x78, "swap", opSwap, asmDefault, disDefault, twoAny, twoAny, 3, modeAny, opSizeDefault},
+	{0x79, "select", opSelect, asmDefault, disDefault, twoAny.plus(oneInt), oneAny, 3, modeAny, opSizeDefault},
+
+	// There must be at least one thing on the stack for dig, but
+	// it would be nice if we did better checking than that.
+	{0x80, "dig", opDig, asmDefault, disDefault, oneAny, twoAny, 3, modeAny, opSize{1, 2, nil}},
+
+	// Like gtxn, but gets txn index from stack, rather than immediate arg
+	{0x81, "stxn", opStxn, assembleStxn, disTxn, oneInt, oneAny, 3, modeAny, opSize{1, 2, nil}},
+	{0x82, "stxna", opStxna, assembleStxna, disTxna, oneInt, oneAny, 3, modeAny, opSize{1, 3, nil}},
+
+	// Immediate bytes and ints. Smaller code size for single use of constant.
```

Comment:
It would be good for the assembler to help optimize this for TEAL 3.
`int` assembler text could become intc_N, intc, or pushint (`inti` ?) for most-used, less-used, used-once
same for `byte`

Reply:
I agree it should. But I'm going to have to delay that to the teal4 (which is the new name for the second half teal3 changes, so is coming by end of Q1).